src/views/containerNode.ts

```diff
@@ -36,7 +36,7 @@ export class ContainerNode extends DataNode {
         return `container/${this.name}`;
     }
 
-    protected get iconPath(): { light: string, dark: string } {
-        return ExplorerNode.resolveIconPath("library");
+    protected get iconPath(): ThemeIcon {
+        return new ThemeIcon("inbox");
```

Comment:
We are actually looking for "books" right? But it's not there. How about " files"?

Reply:
Can we request books in vscode repo?

Reply:
Requested: https://github.com/microsoft/vscode-codicons/issues/7